tools/reviewer/webapp/src/app/dashboard/file-changes/code-changes/code-changes.component.ts

```diff
@@ -106,7 +108,7 @@ export class CodeChangesComponent implements OnInit, OnChanges {
 
   // Add the thread to all threads to display it on user screen
   startThread(thread: Thread): void {
-    const blockIndex: number = this.fileChangesService.getBlockIndex(thread);
+    const blockIndex: number = this.commitService.getBlockIndex(thread);
```

Comment:
I understand you want `blockIndex` so you can for-loop on both sides, right?
How about we create an enum `Side` with `Side.LEFT` and `Side.RIGHT` and that way we can still for loop, but pass the enum here instead of the `blockIndex`?

Reply:
Ok, great